modules/layers/src/geojson-layer/geojson-layer.js

```diff
@@ -157,13 +159,90 @@ export default class GeoJsonLayer extends CompositeLayer {
     const LineStringsLayer = this.getSubLayerClass('line-strings', PathLayer);
     const PointsLayer = this.getSubLayerClass('points', ScatterplotLayer);
 
+    const layerProps = {
```

Comment:
We don't need nested objects for a scoped variable. It will prevent this code from being properly minified. Use

```js
const pointLayerProps = {};
const lineLayerProps = {};
const polygonFillLayerProps = {};
const polygonStrokeLayerProps = {};
```